modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -110,6 +110,10 @@ portletDisplay.setURLBack(backURL);
 					}
 				}
 
+				if (fieldName === 'status') {
+					value = null;
+				}
```

Comment:
Hi @pedro-oliveira446 please do this way at the beginning of the reduce method:

```javascript
				if (field.readOnly) {
					return obj;
				}
```

This way we can avoid magic names like 'status' and we're keeping this logic simple.

Thank you!